pkg/services/store/object/object.proto

```diff
@@ -0,0 +1,140 @@
+syntax = "proto3";
+package object;
+
+option go_package = "./;object";
+
+// Will be replaced with something from the SDK
+message UserInfo {
+  int64 id = 1; // internal grafana ID
+  string login = 2; // string ID? 
+}
+
+// Raw saved object metadata
+message RawObject {
+  // Unique ID for this object
+  string UID = 1;
+
+  // Identify the object kind.  This kind will be used to apply a schema to the body and
+  // will trigger additional indexing behavior.
+  string kind = 2;
+
+  // Time in millies since epoch that the object was first saved
+  int64 created = 3;
+
+  // Time in millies since epoch that the object was last modified
+  int64 updated = 4;
+
+  // Who created the object
+  UserInfo createdBy = 5;
```

Comment:
Protobuf's style convention for field names is snake_case - https://developers.google.com/protocol-buffers/docs/style#message_and_field_names